README.md

```diff
@@ -21,6 +26,9 @@ $ npm run start:dev
 
 # production mode
 $ npm run start:prod
+
+# docker mode
+$ docker-compose up &
```

Comment:
This should be an optional step if somebody has a PG instance and they would want to connect with it.

Reply:
addressed